pyproject.toml

```diff
@@ -23,14 +23,21 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["jwcrypto", "twisted"]
+dependencies = [
+  "jwcrypto",
+  "twisted",
+  "synapse @ git+https://github.com/famedly/synapse.git@v1.103.0_1",
```

Comment:
This is the wrong package name, but also you don't really want to depend on synapse directly. That has caused problems in the past, see https://github.com/famedly/synapse-invite-checker/commit/858a6ed6e856d5cde971e1013ad2cfad0d68a0a8